### PR TITLE
Add visualization graph

### PR DESCRIPTION
**Pull request by leclairm**
Generate an svg graph from the core IR using pygraphviz and adding some interactivity with javascript

NOTE: This requires graphviz to be installed locally

### CODE REVIEW

**Review by DropD — CHANGES_REQUESTED** [review]
While testing it locally with the "small" example configuration, I noticed that the "icon_input" node in the visualization is not connected to anything. Is that supposed to be the case?

**Comment by leclairm** [comment]
> While testing it locally with the "small" example configuration, I noticed that the "icon_input" node in the visualization is not connected to anything. Is that supposed to be the case?

True, I always only look at the large one. It shows the visualization is good check to have! We probably need a PR to well define which scenarios we want to test.

File: src/sirocco/svg-interactive-script.js
[the remark this answers is not in the document]
Comment (leclairm) [reply src/sirocco/svg-interactive-script.js:1]: It's not heavily based it's 99% copied! I didn't see there was a license. Yes I should absolutely include that.

File: src/sirocco/svg-interactive-style.css
[the remark this answers is not in the document]
Comment (leclairm) [reply src/sirocco/svg-interactive-style.css:1]: same as for the `.js` file

File: src/sirocco/svg-interactive-script.js
[the remark this answers is not in the document]
Comment (leclairm) [reply src/sirocco/svg-interactive-script.js:1]: Oh it's actually  brand new. When I "stole" it, it was only one file in the repo. There might be even more to borrow and acknowledge, then.

[the remark this answers is not in the document]
Comment (leclairm) [reply src/sirocco/svg-interactive-script.js:1]: > Might also be not too early now to think about what license the rest of this code should have.
True, I'm happy to receive any input for that, I'm not really used to it.

[the remark this answers is not in the document]
Comment (DropD) [reply src/sirocco/svg-interactive-script.js:1]: I created issue #41 to add a license for the project.

**Review by DropD — APPROVED** [review]

**Pull request merged**